debian/control

```diff
@@ -3,15 +3,46 @@ Section: web
 Priority: optional
 Maintainer: Debian letsencrypt <letsencrypt-devel@lists.alioth.debian.org>
 Uploaders: Francois Marier <francois@debian.org>
-Build-Depends: debhelper (>= 9), python-all, dh-python, python-nose, python-nosexcover, python-tox, python-docutils, python-sphinx, python-sphinx-rtd-theme, python-repoze.sphinx.autointerface
+Build-Depends: debhelper (>= 9),
+               dh-python,
+               python-all,
+               python-configargparse,
+               python-dialog (>=3.2.2rc1),
+               python-docutils,
+               python-nose,
+               python-nosexcover,
+               python-openssl (>= 0.15),
+               python-parsedatetime,
+               python-repoze.sphinx.autointerface,
+               python-rfc3339,
+               python-sphinx,
+               python-sphinx-rtd-theme,
```

Comment:
These are the build dependencies only and so they only affect the package build system (when the docs are generated) and will not be required when admins install the package on their system.